storage-engine/titan-configuration.md

```diff
@@ -105,6 +105,10 @@ If you observe that the Titan GC thread is in full load for a long time from **T
 
 You can adjust [`rate-bytes-per-sec`](/tikv-configuration-file.md#rate-bytes-per-sec) to limit the I/O rate of RocksDB compaction, reducing its impact on foreground read and write performance during high traffic.
 
+### `shared-blob-cache` (New in v8.0.0)
+
+You can control whether to enable the shared cache for Titan blob files and RocksDB block files through [`shared-blob-cache`](/tikv-configuration-file.md#shared-blob-cache-tidb-new-in-v800). The default value is `true`. When the shared cache is enabled, block files have higher priority. This means that TiKV prioritizes meeting the cache needs of block files and then uses the remaining cache for blob files.
+
 ### Titan configuration example
 
```

Comment:
I see that the PR in Chinese also removes the Line 128 `blob-cache-size = "0GB"`.

Reply:
updated in https://github.com/pingcap/docs/pull/16777/commits/1869f2a3570d96d6a66f3afd0ad506b3085ed1a3